[2021-04-04 22:00:41,633] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-04-04T00:00:00+00:00 [queued]>
[2021-04-04 22:00:41,636] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-04-04T00:00:00+00:00 [queued]>
[2021-04-04 22:00:41,636] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-04 22:00:41,636] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-04 22:00:41,636] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-04 22:00:41,649] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): processing_user> on 2021-04-04T00:00:00+00:00
[2021-04-04 22:00:41,652] {standard_task_runner.py:52} INFO - Started process 7691 to run task
[2021-04-04 22:00:41,664] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2021-04-04T00:00:00+00:00', '--job-id', '13', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp1so4f060', '--error-file', '/tmp/tmpfapey11r']
[2021-04-04 22:00:41,664] {standard_task_runner.py:77} INFO - Job 13: Subtask processing_user
[2021-04-04 22:00:41,692] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2021-04-04T00:00:00+00:00 [running]> on host prathamesh-VirtualBox
[2021-04-04 22:00:41,713] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-04-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-04T00:00:00+00:00
[2021-04-04 22:00:41,723] {python.py:118} INFO - Done. Returned value was: None
[2021-04-04 22:00:41,725] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20210404T000000, start_date=20210405T020041, end_date=20210405T020041
[2021-04-04 22:00:41,747] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-04 22:00:41,771] {local_task_job.py:146} INFO - Task exited with return code 0
